File: c/bee1131.c
#include <stdio.h>
 
int main() {
    int golsI, winI=0, golsG, winG=0, draw=0, opc, totalJogos=0;

    while (1) {
        totalJogos++;
        scanf("%d %d", &golsI, &golsG);
        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &opc);
        if (golsI > golsG) {
            winI++;
        } else if (golsI < golsG) {
            winG++;
        } else {
            draw++;
        }
        if (opc == 2) {
            break;
        }
    }

    printf("%d grenais\n", totalJogos);
    printf("Inter:%d\n", winI);
    printf("Gremio:%d\n", winG);
    printf("Empates:%d\n", draw);
    if (golsI > golsG) {
        printf("Inter venceu mais\n");
    } else if (golsI < golsG) {
        printf("Gremio venceu mais\n");
    } else {
        printf("Nao houve vencedor\n");
    }

    return 0;
}

File: beecrowd/python/bee1131.py
cont = i_win = g_win = e = v = 0

while True:
    i, g = map(int, input().split())
    cont += 1
    if i > g:
        i_win += 1
    elif i < g:
        g_win += 1
    else:
        e += 1
    
    if i_win > g_win:
        v = 'Inter venceu mais'
    elif i_win < g_win:
        v = 'Gremio venceu mais'
    else:
        v = 'Nao houve vencedor'
    
    opc = 0
    while True:
        print('Novo grenal (1-sim 2-nao)')
        opc = int(input())
        if opc == 1 or opc == 2:
            break
    if opc == 2:
        break

print(f'{cont} grenais')
print(f'Inter:{i_win}')
print(f'Gremio:{g_win}')
print(f'Empates:{e}')
print(v)
